pkg/constants/constants.go

```diff
@@ -24,4 +24,7 @@ const (
 	IndexURI = "https://github.com/kubernetes-sigs/krew-index.git"
 	// DefaultIndexName is a magic string that's used for a plugin name specified without an index.
 	DefaultIndexName = "default"
+	// EnableMultiIndexFlag is the name of the environment variable that needs to be set to use
+	// the features around multiple indexes (this will be removed later on).
+	EnableMultiIndexFlag = "X_KREW_ENABLE_MULTI_INDEX"
```

Comment:
Is this really a flag? Or is `EnableMultiIndexSwitch` better?
I find  it hard to  come up with a perfect name though. `EnableMultiIndexEnvVarName` is a bit long.

Reply:
I couldn't really come up with a great name either. I think `EnableMultiIndexSwitch` is fine, or what about `EnableMultiIndexEnvVar`?

Reply:
Let's go with switch.